### ISSUE TITLE
Can't connect  -  Connection refused (111)

### ISSUE THREAD

**Issue by win2win** [SocialSend/SocialSend#5]

**Comment by win2win** [comment]
After building the code on Ubuntu 16.04 the client is not syncing. The debug.log files shows this kind of entries.
I have done this build on two different machines with Ubuntu 16.04
I have disabled all firewalls. 

send-cli getconnectioncount returns 0
The same machine works fine with bitcoin which has been also built. 
I have followed this instructions. 
https://steemit.com/socialsend/@edx/instructions-for-creating-master-node-for-social-send-coin

2017-12-16 06:11:13 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:11:53 connect() to 45.77.242.196:50050 failed after select(): Connection refused (111)
2017-12-16 06:12:32 connect() to 45.77.242.196:50050 failed after select(): Connection refused (111)
2017-12-16 06:13:12 connect() to 45.77.242.196:50050 failed after select(): Connection refused (111)
2017-12-16 06:13:54 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:14:34 connect() to 45.77.242.196:50050 failed after select(): Connection refused (111)
2017-12-16 06:15:16 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:15:54 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:16:31 connect() to 45.77.242.196:50050 failed after select(): Connection refused (111)
2017-12-16 06:17:10 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:17:50 connect() to 45.77.242.196:50050 failed after select(): Connection refused (111)
2017-12-16 06:18:32 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:19:12 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:19:55 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:20:37 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:21:17 connect() to 142.91.104.115:50050 failed after select(): Connection refused (111)
2017-12-16 06:21:59 connect() to 45.77.242.196:50050 failed after select(): Connection refused (111)
